frontend/OBSApp.cpp

```diff
@@ -1374,11 +1374,9 @@ string GetFormatString(const char *format, const char *prefix, const char *suffi
 string GetFormatExt(const char *container)
 {
 	string ext = container;
-	if (ext == "fragmented_mp4")
+	if (ext == "fragmented_mp4" || ext == "hybrid_mp4")
 		ext = "mp4";
-	if (ext == "hybrid_mp4")
-		ext = "mp4";
```

Comment:
I feel like this was done this way for a specific reason, but I cannot recall it at present.